modules/effects/src/main/scala/dev/profunktor/redis4cats/runner.scala

```diff
@@ -55,7 +53,7 @@ private[redis4cats] class RunnerPartiallyApplied[F[_]: Concurrent: Log: Parallel
     (Deferred[F, Either[Throwable, w.R]], F.delay(UUID.randomUUID)).tupled.flatMap {
       case (promise, uuid) =>
         def cancelFibers[A](fibs: HList)(err: Throwable): F[Unit] =
-          joinOrCancel(fibs, HNil)(false) >> promise.complete(err.asLeft)
+          joinOrCancel(fibs, HNil)(false) >> promise.complete(err.asLeft).void
```

Comment:
I wonder if we need to assert that these completions return `true`?

Reply:
Hmmm asserting as in test assertions? Or what do you have in mind?

Reply:
Can't do any harm.

Reply:
Sorry, didn't see your response before I answered and resolved this. The code I added should answer your question but we can remove it if you don't think it's needed.